app/components/settings/categories/DisplaySettings.js

```diff
@@ -47,18 +47,18 @@ export default class DisplaySettings extends Component {
     const { theme, selectTheme } = this.props;
     const { intl } = this.context;
 
-    const themeDefaultClasses = classnames([
-      theme === 'themeDefault' ? styles.active : styles.inactive,
+    const themeLightBlueClasses = classnames([
+      theme === 'light-blue' ? styles.active : styles.inactive,
```

Comment:
I think the code would greatly benefit if you refactor the theme identifiers into dedicated constants instead of magic strings. E.g `const THEMES = { LIGHT_BLUE: 'light-blue' }`

Reply:
@DominikGuzei it should be fine now :)
Please give it another check. Thanks!